plugins/trendmicro_visionone/unit_test/test_add_custom_script.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+from icon_trendmicro_visionone.actions import AddCustomScript
+from mock import mock_connection, mock_params
+
+
+class TestAddCustomScript(TestCase):
+    def setUp(self):
+        self.action = AddCustomScript()
+        self.connection = mock_connection()
+        self.action.connection = self.connection
+        self.mock_params = mock_params("add_custom_script")
+
+    def test_1_integration_add_custom_script(self):
+        response = self.action.run(self.mock_params["input"])
+        for key in response.keys():
+            self.assertIn(key, str(self.mock_params["output"].keys()))
```

Comment:
That's ok to use `parameterized` package that allows to parameterise unittests. Please check: https://pypi.org/project/parameterized/

Reply:
<img width="1421" alt="Screenshot 2024-02-07 at 12 15 43" src="https://github.com/rapid7/insightconnect-plugins/assets/131287358/edc2e9e2-0040-46a9-bdeb-48e50a87a5dc">

I would like to confirm whether this will be acceptable before I go in and change all the unit tests to this parameterized syntax.

Reply:
Double checked the way you're doing unittests and in this case please leave them as before (without parameterised). Thanks!